python_tests/pytest_input_generators.py

```diff
@@ -298,6 +298,73 @@ def define_tensor_error_generator(
         yield SampleInput(input_tensor, **es.kwargs), es.ex_type, es.ex_str
 
 
+def define_vector_constant_error_generator(
+    op: OpInfo, dtype: torch.dtype, requires_grad: bool = False, **kwargs
+):
+    """
+    "define_vector",
+    [](FusionDefinition& self, py::list& values) -> Vector {
+    """
+
+    MINIMUM_SYMBOLIC_SIZE = -1
+    INT64_MAX = 9223372036854775807
```

Comment:
Nit: use [sys.maxsize](https://docs.python.org/3/library/sys.html#sys.maxsize):
> An integer giving the maximum value a variable of type Py_ssize_t can take. It’s usually 2**31 - 1 on a 32-bit platform and 2**63 - 1 on a 64-bit platform.

Reply:
Actually nevermind; in this case, hard-coding the int64_t max is better as you are comparing the error from the C++ side..